leader/leader.go

```diff
@@ -140,14 +140,10 @@ func (l *Client) AddWatch(key string, retry time.Duration, valuesC chan string)
 			closer()
 		}()
 
-		backoff := &FreebieExponentialBackOff{
-			InitialInterval:     500 * time.Millisecond,
-			RandomizationFactor: 0.5,
-			Multiplier:          1.5,
-			MaxInterval:         10 * time.Second,
-			MaxElapsedTime:      30 * time.Second,
-		}
+		backoff := NewUnlimitedExponentialBackOff()
+		backoff.MaxInterval = 10 * time.Second
```

Comment:
Would it make sense to include this setting into "NewUnlimitedExponentialBackoff" so when you create a new instance of it, it already comes fully set up?